src/js/consts/valueIndicatorVariant.js

```diff
@@ -1,8 +1,11 @@
-const arrowValueIndicatorVariant = {
+const valueIndicatorVariant = {
   EQUAL: 'EQUAL',
   EMPTY: 'EMPTY',
   POSITIVE: 'POSITIVE',
   NEGATIVE: 'NEGATIVE',
+  MORE: 'MORE',
+  LESS: 'LESS',
+  TRANSACTION: 'TRANSACTION',
```

Comment:
Add a comment where the first 4 values are used right now, where those last 3 are used, I mean generally "what's happening here"